boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -332,3 +333,27 @@ opm-build-push: docker-push
 .PHONY: ensure-fips
 ensure-fips:
 	${CONVENTION_DIR}/configure-fips.sh
+
+.PHONY: migrate-tools
+migrate-tools:
+	# Installing controller-gen v0.8.0
+	go install sigs.k8s.io/controller-tools/cmd/controller-gen@${CONTROLLER_GEN_VERSION}
+	mv ${GOBIN}/controller-gen ${GOBIN}/controller-gen-${CONTROLLER_GEN_VERSION}
+
+	# Installing openapi-gen v0.23.0
+	go install k8s.io/code-generator/cmd/openapi-gen@${OPENAPI_GEN_VERSION}
+	mv ${GOBIN}/openapi-gen ${GOBIN}/controller-gen-${CONTROLLER_GEN_VERSION}
+
+.PHONY: migrate-files
+# You will need to export the forked/cloned operator repository directory as REPO to make this work.
+# Example: export REPO=~/Projects/My-Operator-Fork
+migrate-files:
+	rm -rf config
+	rsync -a $(REPO)/deploy .
+	rsync -a $(REPO)/config/config.go config/
+	rsync -a $(REPO)/config/templates/csv-template.yaml config/templates/
+	rsync -a $(REPO)/pkg/ ./pkg/ --exclude={'apis','controller'}
+	rsync -a $(REPO)/ . --exclude={'cmd','version'} --ignore-existing
```

Comment:
Did you want to copy in boilerplate also?

Reply:
My bad, excluded boilerplate this time.